package.json

```diff
@@ -37,7 +37,7 @@
     "test:babel-plugins": "yarn workspace @glimmer/vm-babel-plugins test",
     "test:browserstack": "ember test --test-port=7774 --host 127.0.0.1 --config-file=testem-browserstack.js",
     "test:lint": "eslint . --quiet",
-    "test:node": "node bin/run-node-tests.mjs",
+    "test:node": "pnpm -r test:node",
```

Comment:
Shouldn't this still go through Turbo? Right now, we only have one node test, but putting it in turbo would allow us to add package-local node tests easily if we wanted to.

Reply:
updated!